Resources/Prototypes/Entities/Markers/pointing.yml

```diff
@@ -10,6 +10,7 @@
     netsync: false
     sprite: Interface/Misc/pointing.rsi
     state: pointing
+    drawDepth: 9
```

Comment:
Should this Just be Overlays? The comment for drawdepth 9 even mentions pointing arrows.

Reply:
Oops, yeah,  it should definitely just be `Overlays`.